lib/instrumentation/index.js

```diff
@@ -15,18 +15,23 @@ module.exports = Instrumentation
 function Instrumentation (agent) {
   var self = this
```

Comment:
seems like travis is not happy as this variable is never used?

Reply:
Fixed